packages/sdk/src/search/Provider.tsx

```diff
@@ -17,7 +17,7 @@ type Props = SearchState & {
   itemsPerPage: number
 }
 
-export const Provider: FC<Props> = ({
+export const Provider: FC<PropsWithChildren<Props>> = ({
```

Comment:
What do you think about this suggestion?

```tsx
export const Provider = ({ ...props /* current object properties*/}: PropsWithChildren<Props>) => { /* current component body here */}
```

Reply:
I like it! Is more clear way. I already changed!